### ISSUE TITLE
TypeError: fit() got an unexpected keyword argument 'verbose'

### ISSUE THREAD

**Issue by wangjianqiao111** [DataCanvasIO/HyperGBM#103]
<em>Please make sure that this is a bug. </em>

**System information**
- OS Platform and Distribution (e.g., CentOS 7.6):linux
- Python version:3.7
- HyperGBM version:0.3.0
- Other Python packages(run `pip list`):
Package                      Version
---------------------------- ------------
absl-py                      1.4.0
accelerate                   0.20.3
aiohttp                      3.8.5
aiosignal                    1.3.1
annotated-types              0.5.0
anyio                        3.7.1
apex                         0.1
appdirs                      1.4.4
argon2-cffi                  23.1.0
argon2-cffi-bindings         21.2.0
astunparse                   1.6.3
async-timeout                4.0.3
asynctest                    0.13.0
attrs                        23.1.0
audioread                    3.0.0
autovizwidget                0.21.0
backcall                     0.2.0
backports.zoneinfo           0.2.1
bcrypt                       4.0.1
beautifulsoup4               4.12.2
bitsandbytes                 0.41.1
bleach                       6.0.0
cachetools                   5.3.1
catboost                     1.2
category-encoders            2.6.2
certifi                      2023.7.22
cffi                         1.15.1
cfgv                         3.3.1
chardet                      5.2.0
charset-normalizer           3.2.0
click                        7.1.2
cloudpickle                  2.2.1
cmake                        3.27.5
colorama                     0.4.6
coloredlogs                  15.0.1
colossalai                   0.3.1
comm                         0.1.4
contexttimer                 0.3.3
crc32c                       2.3.post0
cryptography                 3.4
cycler                       0.11.0
dask                         2022.1.0
dask-glm                     0.2.0
dask-ml                      2022.5.27
dask-sql                     2022.1.0
DataProperty                 1.0.1
datasets                     2.13.1
debugpy                      1.7.0
decorator                    5.1.1
deeptables                   0.2.5
defusedxml                   0.7.1
Deprecated                   1.2.14
dill                         0.3.6
distlib                      0.3.7
distributed                  2022.1.0
docker-pycreds               0.4.0
docutils                     0.20.1
eli5                         0.13.0
entrypoints                  0.4
evaluate                     0.4.0
exceptiongroup               1.1.3
fabric                       3.2.2
fastapi                      0.101.1
fastjsonschema               2.18.0
featuretools                 1.11.1
filelock                     3.12.2
Flask                        1.1.4
Flask-Cors                   3.0.10
Flask-OpenTracing            1.1.0
flatbuffers                  1.12
fonttools                    4.38.0
frozenlist                   1.3.3
fsspec                       2023.1.0
gast                         0.4.0
gin-config                   0.5.0
gitdb                        4.0.10
GitPython                    3.1.36
google-auth                  2.23.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
graphviz                     0.8.4
greenlet                     2.0.2
grpcio                       1.58.0
grpcio-opentracing           1.1.4
grpcio-reflection            1.34.1
gssapi                       1.8.3
gunicorn                     19.9.0
h11                          0.14.0
h5py                         3.8.0
hdijupyterutils              0.21.0
HeapDict                     1.0.1
holidays                     0.27.1
huggingface-hub              0.16.4
humanfriendly                10.0
hypergbm                     0.3.0
hypernets                    0.3.0
icetk                        0.0.7
identify                     2.5.24
idna                         3.4
imbalanced-learn             0.11.0
importlib-metadata           6.7.0
importlib-resources          5.12.0
invoke                       2.2.0
ipykernel                    5.3.0
ipython                      7.34.0
ipython-genutils             0.2.0
ipywidgets                   8.1.1
itsdangerous                 1.1.0
jaeger-client                4.1.0
jaraco.classes               3.2.3
jedi                         0.19.0
jeepney                      0.8.0
jieba                        0.42.1
Jinja2                       2.11.3
joblib                       1.3.2
JPype1                       1.4.1
jsonlines                    3.1.0
jsonschema                   4.17.3
jupyter                      1.0.0
jupyter_client               7.4.9
jupyter-console              6.6.3
jupyter_core                 4.12.0
jupyter-server               1.24.0
jupyterlab-pygments          0.2.2
jupyterlab-widgets           3.0.9
keras                        2.11.0
Keras-Applications           1.0.8
Keras-Preprocessing          1.1.2
keyring                      24.1.1
kiwisolver                   1.4.5
krb5                         0.5.1
libclang                     16.0.6
librosa                      0.9.2
lightgbm                     4.0.0
lit                          16.0.6
llvmlite                     0.39.1
lm-eval                      0.3.0
locket                       1.0.0
Markdown                     3.4.4
markdown-it-py               2.2.0
MarkupSafe                   2.1.3
matplotlib                   3.5.3
matplotlib-inline            0.1.6
mbstrdecoder                 1.1.3
mdurl                        0.1.2
mistune                      3.0.1
more-itertools               9.1.0
mpmath                       1.3.0
msgpack                      1.0.5
multidict                    6.0.4
multipledispatch             1.0.0
multiprocess                 0.70.14
mxnet                        1.9.1
nbclassic                    1.0.0
nbclient                     0.7.4
nbconvert                    7.6.0
nbformat                     5.8.0
nest-asyncio                 1.5.5
ninja                        1.11.1
nltk                         3.8.1
nodeenv                      1.8.0
notebook                     6.5.5
notebook_shim                0.2.3
numba                        0.56.4
numexpr                      2.8.6
numpy                        1.21.6
nvidia-ml-py3                7.352.0
oauthlib                     3.2.2
onnx                         1.12.0
onnxruntime                  1.14.1
openai                       0.28.0
opencv-python                4.8.0.76
opentracing                  2.2.0
opt-einsum                   3.3.0
optimum                      1.11.1
packaging                    23.1
pandas                       1.3.5
pandasql                     0.7.3
pandocfilters                1.5.0
paramiko                     3.3.1
parso                        0.8.3
partd                        1.4.0
pathtools                    0.1.2
pathvalidate                 3.2.0
patsy                        0.5.3
peft                         0.3.0
pexpect                      4.8.0
pickleshare                  0.7.5
pika                         1.3.2
Pillow                       9.5.0
pip                          23.0.1
pkginfo                      1.9.6
pkgutil_resolve_name         1.3.10
platformdirs                 3.10.0
plotly                       5.17.0
pooch                        1.7.0
portalocker                  2.6.0
pre-commit                   2.21.0
prettytable                  3.7.0
prometheus-client            0.7.1
prompt-toolkit               3.0.39
protobuf                     3.12.2
psutil                       5.8.0
ptyprocess                   0.7.0
py4j                         0.10.9.7
pyaml                        23.5.8
pyarrow                      12.0.1
pyasn1                       0.5.0
pyasn1-modules               0.3.0
pybind11                     2.11.1
pycocotools                  2.0.7
pycountry                    22.3.5
pycparser                    2.21
pydantic                     2.3.0
pydantic_core                2.6.3
pyee                         10.0.2
Pygments                     2.16.1
pygraphviz                   1.7
PyNaCl                       1.5.0
pyparsing                    3.1.1
pyrsistent                   0.19.3
pyspark                      3.4.1
pyspnego                     0.9.2
pytablewriter                1.0.0
python-dateutil              2.8.2
python-ffmpeg                2.0.4
pytz                         2023.3.post1
PyYAML                       5.4
pyzmq                        24.0.1
qtconsole                    5.4.4
QtPy                         2.4.0
ray                          2.6.3
readme-renderer              37.3
redis                        5.0.0
regex                        2023.8.8
requests                     2.31.0
requests-kerberos            0.14.0
requests-oauthlib            1.3.1
requests-toolbelt            1.0.0
resampy                      0.4.2
responses                    0.18.0
rfc3986                      2.0.0
rich                         13.5.3
rouge-score                  0.1.2
rsa                          4.9
rwkv                         0.8.9
sacrebleu                    1.5.0
safetensors                  0.3.2
scikit-learn                 1.0.2
scikit-optimize              0.9.0
scikit-plot                  0.3.7
scipy                        1.7.3
seaborn                      0.12.2
SecretStorage                3.3.3
seldon-core                  1.10.0
semantic-version             2.10.0
Send2Trash                   1.8.2
sentencepiece                0.1.99
sentry-sdk                   1.31.0
setproctitle                 1.3.2
setuptools                   68.0.0
setuptools-rust              1.7.0
shap                         0.42.1
six                          1.16.0
sklearn-pandas               2.2.0
sklearn2pmml                 0.97.2
smmap                        5.0.1
sniffio                      1.3.0
sortedcontainers             2.4.0
soundfile                    0.12.1
soupsieve                    2.4.1
sparkmagic                   0.20.5
SQLAlchemy                   2.0.20
sqlitedict                   2.1.0
starlette                    0.27.0
statsmodels                  0.13.5
sympy                        1.10.1
tabledata                    1.3.3
tables                       3.7.0
tabular-toolbox              0.1.4
tabulate                     0.9.0
tblib                        2.0.0
tcolorpy                     0.1.4
tenacity                     8.2.3
tensorboard                  2.11.2
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorboardX                 2.6
tensorflow                   2.11.0
tensorflow-addons            0.13.0
tensorflow-estimator         2.11.0
tensorflow-hub               0.11.0
tensorflow-io-gcs-filesystem 0.34.0
tensorflow-text              2.11.0
tensorrt                     8.6.1
termcolor                    2.3.0
terminado                    0.17.1
tf-models-official           2.11.6
tf2onnx                      1.14.0
tfrecord                     1.14.3
threadloop                   1.0.2
threadpoolctl                3.1.0
thrift                       0.16.0
timm                         0.9.5
tinycss2                     1.2.1
tokenizers                   0.13.3
tomli                        2.0.1
toolz                        0.12.0
torch                        1.13.1+cu116
torchaudio                   0.13.1+cu116
torchtext                    0.14.1
torchvision                  0.14.1+cu116
tornado                      5.1.1
tqdm                         4.66.1
tqdm-multiprocess            0.0.11
traitlets                    5.9.0
transformers                 4.30.2
triton                       2.0.0
twine                        4.0.2
typeguard                    2.13.3
typepy                       1.3.1
typing_extensions            4.7.1
tzlocal                      5.0.1
urllib3                      1.25.9
uvicorn                      0.22.0
virtualenv                   20.24.3
wandb                        0.15.8
wcwidth                      0.2.6
webencodings                 0.5.1
websocket-client             1.6.1
Werkzeug                     1.0.1
wheel                        0.41.2
widgetsnbextension           4.0.9
woodwork                     0.16.4
wordcloud                    1.5.0
wrapt                        1.15.0
xgboost                      1.6.2
XlsxWriter                   3.1.4
xxhash                       3.3.0
yarl                         1.9.2
zict                         2.2.0
zipp                         3.15.0
zstandard                    0.21.0

**
2023-10-19 14:04:06.086 [ERROR] 10-19 14:04:06 E hypernets.m.hyper_model.py 103 - run_trail failed! trail_no=1
2023-10-19 14:04:06.090 [ERROR] 10-19 14:04:06 E hypernets.m.hyper_model.py 105 - Traceback (most recent call last):
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/hypernets/model/hyper_model.py", line 98, in _run_trial
2023-10-19 14:04:06.090 [ERROR]     estimator.fit(X, y, **fit_kwargs)
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/hypergbm/hyper_gbm.py", line 459, in fit
2023-10-19 14:04:06.090 [ERROR]     self.model.fit(X, y, **fit_kwargs)
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/hypergbm/dask/_estimators.py", line 43, in fit
2023-10-19 14:04:06.090 [ERROR]     super(LGBMClassifierDaskWrapper, self).fit(X, y, sample_weight=sample_weight, **kwargs)
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/lightgbm/dask.py", line 1214, in fit
2023-10-19 14:04:06.090 [ERROR]     **kwargs
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/lightgbm/dask.py", line 1099, in _lgb_dask_fit
2023-10-19 14:04:06.090 [ERROR]     **kwargs
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/lightgbm/dask.py", line 818, in _train
2023-10-19 14:04:06.090 [ERROR]     results = client.gather(futures_classifiers)
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/distributed/client.py", line 1952, in gather
2023-10-19 14:04:06.090 [ERROR]     asynchronous=asynchronous,
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/distributed/utils.py", line 311, in sync
2023-10-19 14:04:06.090 [ERROR]     self.loop, func, *args, callback_timeout=callback_timeout, **kwargs
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/distributed/utils.py", line 364, in sync
2023-10-19 14:04:06.090 [ERROR]     raise exc.with_traceback(tb)
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/distributed/utils.py", line 349, in f
2023-10-19 14:04:06.090 [ERROR]     result[0] = yield future
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/tornado/gen.py", line 1133, in run
2023-10-19 14:04:06.090 [ERROR]     value = future.result()
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/distributed/client.py", line 1811, in _gather
2023-10-19 14:04:06.090 [ERROR]     raise exception.with_traceback(traceback)
2023-10-19 14:04:06.090 [ERROR]   File "/usr/local/python3.7.17/lib/python3.7/site-packages/lightgbm/dask.py", line 322, in _train_part
2023-10-19 14:04:06.090 [ERROR]     **kwargs
2023-10-19 14:04:06.090 [ERROR] TypeError: fit() got an unexpected keyword argument 'verbose'
**

**Describe the expected behavior**

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Jupyter notebook.

**Are you willing to submit PR?(Yes/No)**

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. 
If including tracebacks, please include the full traceback. Large logs and files 
should be attached.

**Comment by wangjianqiao111** [comment]
program code：

from dask import dataframe as dd
from dask.distributed import LocalCluster, Client

from hypergbm import make_experiment
from hypergbm.dask import search_space_general
from hypernets.tabular.datasets import dsutils


def my_search_space():
    return search_space_general(n_estimators=100)


def train():
    cluster = LocalCluster(processes=False)
    client = Client(cluster)

    train_data = dd.from_pandas(dsutils.load_blood(), npartitions=1)

    experiment = make_experiment(train_data, target='Class', searcher='mcts', search_space=my_search_space)
    estimator = experiment.run()
    print(estimator)


if __name__ == '__main__':
    train()

**Comment by oaksharks** [comment]
Please try downgrading the version of lightgbm or using hypergbm on the main branch.
